client/stylesheets/shared/_embed.scss

```diff
@@ -125,3 +125,17 @@
 		}
 	}
 }
+
+// for the embed page with the new navigation enabled
+.woocommerce-feature-enabled-navigation.woocommerce-embed-page {
+	@include breakpoint( '<782px' ) {
+		#screen-meta,
+		#screen-meta-links {
+			top: $header-height-with-new-nav;
```

Comment:
This probably doesn't quite work when the header height changes, such as when the mobile app banner exists.

Reply:
@joshuatf Thank you! In that case..I will wait for #6247 to get tested and merged before making any further changes in this branch.